docs/UserGuide.md

```diff
@@ -164,6 +170,28 @@ Format: `unremind INDEX`
 * The index refers to the index number shown in the displayed assignment list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
+### Setting priority for assignments : `prioritize`
+
+Sets a priority tag for the specified assignment.
+
+Format: `prioritize INDEX priority/PRIORITY`
+
+* Sets the priority to the assignment at the specified `INDEX`.
+* Priority levels include LOW, MEDIUM and HIGH.
+* The index refers to the index number shown in the displayed assignment list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* If the assignment already has a priority tag, this command will replace the previous priority tag with the new one.
+
+### Removing priority for assignments : `unprioritize`
+
+Sets a priority tag for the specified assignment.
+
+Format: `prioritze INDEX`
+
```

Comment:
The description and format does not fit the unprioritize command :)

Reply:
Amended.